### ISSUE TITLE
[Feature Request] Rename "Advance" to "Prepone"

### ISSUE THREAD

**Issue by Expertium** [open-spaced-repetition/fsrs4anki-helper#417]
https://www.reddit.com/r/Anki/comments/1attbo1/comment/l4pdbc3/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button

> I think Advance is confusing because it usually means advancing to forward direction and English speakers generally consider the Future to be in that direction. I suggest using the word prepone. It's a bit rare to see that word but it's real and also intuitive.

Also @user1823

**Comment by L-M-Sherlock** [comment]
I don't know this word until today.

**Comment by user1823** [comment]
> I don't know this word until today.

I use this word regularly in communication, but it's probably because I live in India.

![image](https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/92206575/bd2ef6ab-25cd-4c8a-a347-160dffe77e7f)

I am not sure how commonly it is used outside India.

**Comment by L-M-Sherlock** [comment]
<img width="701" alt="image" src="https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/32575846/ea81a74f-9624-48b3-8651-4508562410f1">

Lmao.

**Comment by Expertium** [comment]
To be honest, I didn't know the word "prepone" either.

**Comment by L-M-Sherlock** [comment]
Me too.

**Comment by brishtibheja** [comment]
I was the person who suggested this word. I think a case can be made that prepone is pretty intuitive when you already have the word postpone in the back of your mind. It is not really about knowing the word.

I think prepone is a word you shouldn't use on it's own (unless you live in India) but just from my personal experiences I see people instantly understand the word when the context is right. I'll give it a bit more thought.

**Comment by brishtibheja** [comment]
> I use this word regularly in communication, but it's probably because I live in India.

@user1823 Funny thing is I don't even use it and never see normal people in NE India use it. Learned it in some interview of Sashi Tharoor. It's formal and stiff in my mind and not so common. 

BUT but that doesn't mean it shouldn't be used. Optimiser/Scheduler are rarer words too albeit they make sense. Prepone makes no sense to most people UNTIL they see the word postpone very close to it. So IMO it will be more intuitive than Advance (which literally doesn't make sense plus not many people are familiar with SM so most of us aren't seeing a familiar term here).

**Comment by Expertium** [comment]
1) According to https://datayze.com/word-analyzer, postpone is the 6663rd most common word in the English language. Prepone doesn't even have a rank, meaning that this database doesn't have it.
![image](https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/83031600/0fdfa2f9-25af-4d98-816f-e97b1ef10a84)
![image](https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/83031600/73ecebd9-73e6-40a2-a1d0-7788a154788b)

2) According to https://frequencylist.com/, postpone is the 5035th most common word in the English language. Prepone wasn't found.
![image](https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/83031600/fec8348c-e633-4739-bfe3-8b4bb45276e1)
![image](https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/83031600/564af2dc-8f52-407c-a40b-26f02a131b76)


3) According to one of my Anki decks with frequencies, it is the 5612th most common word. My Anki deck doesn't have prepone.

I think this is fairly conclusive evidence that most people will not be familiar with "prepone".

**Comment by user1823** [comment]
TL;DR
I suggest keeping the word "Advance", just because it is more commonly used than "Prepone".

<hr>

> @user1823 Funny thing is I don't even use it and never see normal people in NE India use it. 

I live in Delhi and "prepone" is as commonly used as "postpone". People use it, newspapers use it and (I think) the government uses it too.

> BUT that doesn't mean it shouldn't be used. 

It does. It is desirable to use a word that is frequently encountered by most people, unless no such word exists. The word "prepone" doesn't fit into the category.

> Advance (which literally doesn't make sense

It does. Here is the definition of "prepone" from Google Dictionary (based on Oxford English Dictionary). See how it uses "advance" as the synonym of "prepone".

> to change the time or date of an event so that it takes place earlier; to advance something

Lastly, here are some quotes from Quora and Stack Exchange:

> If you are writing for a global or formal audience, it is generally best to use "advance" or "bring forward" instead of "prepone" to ensure better clarity and acceptance.

> One correction from an American… while “advance” may be the technically correct term for what you say as “prepone,” the vast majority of Americans simply say “move up.”
>
> “The meeting that was scheduled for Thursday was moved up to today.”

> Just on a whim I checked the dictionary, and it turns out that prepone is in the dictionary! I might personally still avoid it just because many people might be confused by it (or at least take a moment to parse it), but technically it is a perfectly cromulent word.

**Comment by brishtibheja** [comment]
> I think this is fairly conclusive evidence that most people will not be familiar with "prepone".

@Expertium Read again what I said and I very clearly mention most people literally have never heard this weird unless they're Indians. There's no argument there. I was merely claiming, from personal experience, most people don't need an explanation for prepone when it comes along with the word postpone. Exactly because postpone is just so common changing up the prefixes doesn't create problems for people (it can though, as I already said, if they don't have the context). I was very clear about what I was trying to say. Adding modifying affixes don't make things less understandable for native speakers. I mean there's a reason enshittification is such a popular term (people "get" the word instantly).

**Comment by brishtibheja** [comment]
> If you are writing for a global or formal audience, it is generally best to use "advance" or "bring forward" instead of "prepone" to ensure better clarity and acceptance.

@user1823 If most L1 speakers say Advance then it's fine ig. My whole point was people might get confused with that word. If that's not the case outside India then it's not an issue. I thought it could be a potential improvement so could not help but bring it up. Sorry guys, wasted valuable time of everyone. Let's end this here.
